sf-stream/src/protos/extractor.proto

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+message Transaction {
+  uint64 timestamp = 1;
+  uint64 version = 2;
+  TransactionInfo info = 3;
+  uint64 epoch = 4;
+  uint64 block_height = 5;
+
+  enum TransactionType {
+    GENESIS = 0;
+    BLOCK_METADATA = 1;
+    STATE_CHECKPOINT = 2;
+    USER = 3;
+  }
+  
+  TransactionType type = 6;
+
+  oneof txn_data {
+    BlockMetadataTransaction block_metadata_txn = 7;
```

Comment:
`_txn` suffixes should be removed, don't think they add great value and in generated code, in some languages like Go and C++, the values are already prefixed with the message type:

```
type Transaction_BlockMetadataTxn struct {
	BlockMetadataTxn *BlockMetadataTransaction `protobuf:"bytes,7,opt,name=block_metadata_txn,json=blockMetadataTxn,proto3,oneof"`
}
```

So they `Txn` is really redundant.